src/rpc/mining.cpp

```diff
@@ -1187,7 +1200,7 @@ static const CRPCCommand commands[] =
 
     { "generating",         "generatetoaddress",      &generatetoaddress,      {"nblocks","address","maxtries"} },
     { "generating",         "generatetodescriptor",   &generatetodescriptor,   {"num_blocks","descriptor","maxtries"} },
-    { "generating",         "generateblock",          &generateblock,          {"output","transactions"} },
+    { "generating",         "generateblock",          &generateblock,          {"output","transactions","submit"} },
```

Comment:
Also should be added to "src/rpc/client.cpp" to be parsed as non-string on -cli, right?

Reply:
Doh. Can't wait until RPCMan does that for me

#18531

Reply:
oh nice, will review

Reply:
Thanks, fixed.